tools/rna_tools/targetfinder/TargetFinder_threads.xml

```diff
@@ -0,0 +1,77 @@
+<tool id="targetfinder_threads" name="Plant small RNA target prediction tool" version="0.1.0">
+    <requirements>
+        <requirement type="package" version="1.07">targetfinder</requirement>
+        <requirement type="package" version="35.4.12">fasta</requirement>
+    </requirements>
+    <stdio>
+        <exit_code range="1:" />
+    </stdio>
+
+    <command><![CDATA[
+        targetfinder_threads.pl
+            -f $f
+            -d $d
+            -c $c
+            -t "\${GALAXY_SLOTS:-12}"
+            $r
+            -o $output1
+    ]]></command>
+    <inputs>
+        <param label="Input small RNA sequences file" help="(-f) FASTA-format" name="f" type="data" format="fasta" />
```

Comment:
thanks for the cool tip :) 

yet, I like the current configuration better since it is more transparent to me while passing arguments.